src/SWP/Bundle/ContentBundle/Controller/RoutesController.php

```diff
@@ -237,7 +238,7 @@ private function handleRouteUpdate($route, $routeData)
             $route->setName($routeData['name']);
         }
 
-        if (isset($routeData['type']) && $routeData['type'] == Document\Route::TYPE_CONTENT) {
+        if (isset($routeData['type']) && $routeData['type'] == Phpcr\Route::TYPE_CONTENT) {
```

Comment:
This is for now so the app can work, ultimately all calls to phpcr specific models will be handled by the manager service which will be used based on configured storage. This route call will go to model, right. I will clean this up when I will push manager implementation.